devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusInfo.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.gradle.tasks;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+
+import io.quarkus.devtools.commands.data.QuarkusCommandInvocation;
+import io.quarkus.devtools.commands.data.QuarkusCommandOutcome;
+import io.quarkus.devtools.commands.handlers.InfoCommandHandler;
+import io.quarkus.devtools.commands.handlers.UpdateCommandHandler;
+import io.quarkus.devtools.project.QuarkusProject;
+
+public class QuarkusInfo extends QuarkusPlatformTask {
+
+    private boolean perModule = false;
+
+    @Input
+    public boolean getPerModule() {
+        return perModule;
+    }
+
+    @Option(description = "Log project's state per module.", option = "perModule")
```

Comment:
didn't you want to hide/remove this option for this round?

agree on its purpose, just not sold on its name :)